core/modules/views/plugins/views_plugin_query_default.inc

```diff
@@ -170,7 +159,7 @@ class views_plugin_query_default extends views_plugin_query {
    * based on $pure_distinct.
    *
    * @param bool $value
-   *   Should the view by distincted.
+   *   Should the view by distinct.
```

Comment:
```suggestion
   *   Should the view be distinct?
```

Reply:
Same as my previous comment, should we use "Whether" here instead of asking a question?

Reply:
Since it's declared as a bool, saying "TRUE or FALSE" is a bit unnecessary. It could be made clearer if we changed the variable name. Your point about "Whether" is good. I suggest:

```
   /**
   * Set the view to be distinct.
   *
   * There are either distinct per base field or distinct in the pure sql way,
   * based on $pure_distinct.
   *
   * @param bool $is_distinct
   *   Whether the view is distinct.
   * @param bool $pure_distinct
   *   Whether only the sql keyword be added.
   */
  function set_distinct($is_distinct = TRUE, $pure_distinct = FALSE) {
    if (!(isset($this->no_distinct) && $is_distinct)) {
      $this->distinct = $is_distinct;
      $this->pure_distinct = $pure_distinct;
    }
  }
```

Reply:
@bugfolder I don't agree with changing the variable name here. For a function called `set_distinct()`, I think the variable name `$value` makes sense (because it's the value you're setting).

Reply:
OK. The rename better sense to me, but you did the heavy lifting on this one (really heavy!), so I can live with it.